excel/tests/test.bal

```diff
@@ -0,0 +1,383 @@
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/os;
+import ballerina/lang.runtime;
+import ballerina/log;
+import ballerina/test;
+
+configurable string clientId = os:getEnv("CLIENT_ID");
+configurable string clientSecret = os:getEnv("CLIENT_SECRET");
+configurable string refreshToken = os:getEnv("REFRESH_TOKEN");
+configurable string refreshUrl = os:getEnv("REFRESH_URL");
+configurable string workbookIdOrPath = os:getEnv("WORKBOOK_PATH");
+
+ExcelConfiguration configuration = {
+    authConfig: {
+        clientId: clientId,
+        clientSecret: clientSecret,
+        refreshToken: refreshToken,
+        refreshUrl: refreshUrl
+    }
+};
+
+Client excelClient = check new (configuration);
+string workBookId = workbookIdOrPath;
+string worksheetName = "testSheet";
+string tableName = "";
```

Comment:
In constants.bal, there is already constant defined as `EMPTY_STRING `
But in this test.bal, still `""` is used in many places.

Shall we use `EMPTY_STRING ` in all places?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/commit/5faa81d3bd0c91f10272734f80b7d29c9ee53f63